src/App.tsx

```diff
@@ -1,5 +1,19 @@
+import { Outlet } from 'react-router-dom';
+
+import styles from './App.module.scss';
+import { Container } from './components/Container';
```

Comment:
Wouldn't it be better to create an `index.ts` file in the components folder and import components outside this folder from it?

```suggestion
import { Container } from './components';
```